src/main/core/support/units.rs

```diff
@@ -0,0 +1,703 @@
+use std::convert::TryInto;
+use std::fmt::{self, Debug, Display};
+use std::str::FromStr;
+
+use once_cell::sync::Lazy;
+use regex::Regex;
+use serde::de::{Deserialize, Deserializer, Visitor};
+use serde::ser::{Serialize, Serializer};
+
+pub trait Prefixes: Clone + Copy + Default + PartialEq + FromStr + Display + Debug {
```

Comment:
Nit throughout - it's more conventional to name a trait, type, or enum after a member of that type. e.g. `Prefix`, `SiPrefix`

Reply:
Fixed